ui/src/app/cohort-search/overview/overview.component.html

```diff
@@ -162,3 +181,28 @@ <h2>Save New Cohort</h2>
     </button>
   </div>
 </clr-modal>
+<clr-modal #deleteModal
```

Comment:
I think you can use the confirmDeleteModal here: https://github.com/all-of-us/workbench/tree/master/ui/src/app/views/confirm-delete-modal 

Just want to make sure we aren't redoing code paths/getting out of sync stylistically. 

Example usage here: https://github.com/all-of-us/workbench/blob/master/ui/src/app/views/resource-card/component.tsx#L414

Reply:
I did look at that component first and it would work but it didn't look like there was a way to set the custom message specified in the ticket (at least not without making changes to the component).

Reply:
I think we will want to support this verbiage change for all cohort deletion, so it probably makes sense to change that component. (To either support custom language, or change the delete language for cohorts)

Reply:
Good call. I didn't think about deleting cohorts from the card having a different message.

I moved confirmDeleteModal to the cohort-common.module so I could use it in cohort-search.